metaflow/plugins/kfp/kfp_step_decorator.py

```diff
@@ -0,0 +1,32 @@
+from metaflow.decorators import StepDecorator
+from metaflow.exception import MetaflowException
+
+
+class KfpStepDecorator(StepDecorator):
+    """
+    For KFP orchestrator plugin only.
+
+    Step decorator to specify the image for the step.
+
+    To use, follow the example below.
+    ```
+    @kfp_step(image="image_on_dockerhub")
+    @step
+    def start(self):
+        self.next(self.next_step)
+
+    ```
+
+    Parameters
+    ----------
+    image: str
+        Defaults to None, which means default to base image.
+        Must resolve to an actual image, either publicly hosted
+        or available for download on the customer's infra.
+    """
+
+    name = "kfp_step"
```

Comment:
Can you try this name with `"kfp"`? 
https://zillowgroup.slack.com/archives/C020U025QJK/p1632938974058100?thread_ts=1632935029.055800&cid=C020U025QJK

Reply:
Makes sense! Updated this PR to make use of the existing `@kfp` decorator`.

Reply:
Second this name change

Not for this PR necessarily but I found this line in conda decorator points to a "kfp_internal" deco name that is no longer used.

Reply:
I'd say let's not do that in this PR considering the time sensitivity.